src/myrtlespeech/data/dataset/composite.py

```diff
@@ -0,0 +1,73 @@
+from typing import Tuple
+
```

Comment:
This `Composite` wrapper currently lacks `docs` documentation.

Reply:
Added